### ISSUE TITLE
Cannot log in

### ISSUE THREAD

**Issue by josephmasson** [illl48/RobInDaHood#9]
Is the app down now? Im very interested in using it but I cannot get past the login screen. If i exit the app, I have to uninstall and redownload in order to get to the login screen. Any updates or anything Im missing?

**Comment by perfectra1n** [comment]
I don't think that he is supporting it anymore because there is an online app that robinhood now officially supports.
